streams/src/main/java/org/apache/kafka/streams/kstream/Consumed.java

```diff
@@ -96,13 +112,21 @@ protected Consumed(final Consumed<K, V> consumed) {
      *
      * @return a new instance of {@link Consumed}
      */
+    @Deprecated
```

Comment:
Also need to add corresponding Java tag.

Reply:
Added, thank you